File: Homework/CSharp-Part-1/03. Operators-and-Expressions/ThirdDigit.cs
using System;

class ThirdDigit
{
    static void Main()
    {
        int number = int.Parse(Console.ReadLine());
        int thirdDigit = (number / 100) % 10;
        int targetDigit = 7;
        Console.WriteLine((thirdDigit == targetDigit) ?
                          "true" : 
                          String.Format("false {0}", thirdDigit));
    }
}

File: Homework/JavaScript-Fundamentals/04. Operators and Expressions/third-digit.js
function solve(args) {
    // ThirdDigit

    // const args = ['9999799'];

    var number = +args[0];

    var thirdDigit = (Math.floor(number / 100)) % 10;
    var targetDigit = 7;

    console.log((thirdDigit == targetDigit) ?
        'true' :
        'false ' + thirdDigit);
}